addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -10,20 +10,22 @@ var QWeb = core.qweb;
 
 models.PosModel = models.PosModel.extend({
     create_printer: function (config) {
-        var url = config.proxy_ip || '';
-        if(url.indexOf('//') < 0) {
-            url = window.location.protocol + '//' + url;
-        }
-        if(url.indexOf(':', url.indexOf('//') + 2) < 0 && window.location.protocol !== 'https:') {
-            url = url + ':8069';
+        if (!config.device_identifier && config.printer_type == 'iot'){
```

Comment:
Is this condition really necessary? I think this function will only be called if it wasn't an epson printer or an iot device

Reply:
I see only one case, when it can be usefull (but probable nearly never), is that you configure an epson printer, and the you uninstall the module. It will leave a record, with the printer_type==='epson', that will arrive here

Reply:
But then this function won't return anything and it will crash on line 43

Reply:
@aprieels allright